test/fixtures/output/curl/application-json.sh

```diff
@@ -1,4 +1,4 @@
 curl --request POST \
   --url "http://mockbin.com/har" \
-  --header "content-type: application/json" \
-  --data "{\"foo\": \"bar\"}"
+  --header "Content-Type: application/json" \
```

Comment:
please rebase, all current output fixtures are functional and working, this will break them (headers are lower cased now)